### ISSUE TITLE
`yq` is a hard requirement for `make`

### ISSUE THREAD

**Issue by hackeryarn** [getoutreach/stencil#214]
Running `make` on a fresh build requires `yq` to even pull down `devbase`. There is an `asdf` plugin for `yq`, but it installs the Go `yq` and not the Python `yq` (and I believe that the Python `yq` is expected). 

In the short term, this hard dependency should be documented. For the long term, should this be an be an `asdf` plugin of its own, or should the project move to the Go `yq` which is much more portable and doesn't require Python.

**Comment by jaredallard** [comment]
Agreed, we [actually have a wrapper to support both](https://github.com/getoutreach/devbase/blob/main/templates/scripts/devbase.sh.tpl#L10) but it's used _only_ in limited cases. This one definitely belongs in devbase though, so I'm going to move it there.

My one issue with the go yq is that it decided to make it's own interface instead of keeping what `jq` had. Very sad about that.
